storage/shard.go

```diff
@@ -1699,12 +1718,14 @@ func (s *dbShard) Flush(
 
 	var multiErr xerrors.MultiError
 	tmpCtx := context.NewContext()
+
+	numBlockDoesNotExist := 0 // Debug
```

Comment:
same here, could you make a separate type `dbShardFlushOutcome` or some such

Reply:
done